### ISSUE TITLE
Feature Request: Find

### ISSUE THREAD

**Issue by DBKJM** [notepad-plus-plus/notepad-plus-plus#3445]
When the Find feature has reached the end of its searches, it blinks the window title bar and menu titles.

When using the F3 button to repeat searching throughout a window, this can be very easy to miss and you end up going round and round the same collection.

Please could you make it more obvious when you've reached the end of the loop? Perhaps blink the whole window in some way.

Many thanks.

**Comment by richlux** [comment]
I agree.  I like what TextPad does, which is an audible "ding" when you hit the end.

Rich

**Comment by MattHeffron** [comment]
I agree with the problem and need for better indication of completion of all of the searches.
However, an audible indication doesn't work well in a "cubicle farm" where all of the audio is silenced. :-(

**Comment by alankilborn** [comment]
>Perhaps blink the whole window in some way.

I'd suggest doing what the Pythonscript command `editor.flash()` does.
Of course, I don't know if it is worth it to install PS just to demo this effect. :-)
But basically, it inverts the video of the editing window for a very short time before restoring it -- very noticeable.

**Comment by anthonyleestark** [comment]
> I agree. I like what TextPad does, which is an audible "ding" when you hit the end.
> 
> Rich

This sounds cool.
But N++ has an option to mute all sounds, I think only an audible "ding" is not enough.
I like Visual Studio's approach, which mixes an audible "ding" and a message box saying "No more occurrences found in the specified documents" (which can also be disabled by unchecking "Always show this message")
If anyone wants the original behavior, turn both off =)))

**Comment by alankilborn** [comment]
>and a message box saying "No more occurrences found

Ugh; no msg box please.

**Comment by anthonyleestark** [comment]
> Ugh; no msg box please.

Ugh, I said to make the message box optional.
When enabled, a message box will be displayed instead of showing information in the Find dialog's status bar. This is helpful if the sound is muted or if the Find dialog is not visible or appears transparent.
You can disable it in `Preferences > Searching` (disabled by default)

**Comment by anthonyleestark** [comment]
> Ugh; no msg box please.

@alankilborn 
Ugh, I said to make the message box optional.
When enabled, a message box will be displayed instead of showing information in the Find dialog's status bar. This is helpful if the sound is muted or if the Find dialog is not visible or appears transparent.
You can disable it in `Preferences > Searching` (disabled by default)

**Comment by alankilborn** [comment]
>Ugh, I said to make the message box optional.

In case you haven't been around long enough, the author of Notepad++ really dislikes adding new preference settings for everything.

**Comment by donho** [comment]
@alankilborn 
Do we have any similar request?

**Comment by alankilborn** [comment]
>...sounds...

Sure, but as pointed out earlier, there are more visual cues than sounds.
And, probably the biggest cue that there is no further search match:  The caret does not move and no next-match selection is made!

**Comment by donho** [comment]
This issue was created 8 years ago, there has been no indication message on Find dialog yet.
The FR is obsolete.
